build/Makefile

```diff
@@ -52,7 +50,7 @@ DEBUG_FLAGS := -g
 endif
 
 # Need to use RUBY_COMPILE_FLAGS when compiling code that uses C Ruby interfaces
-# (Note that "-Wshadow -Wredundant-decls" don't work with Ruby v2.2.0)
+# (Note that "-Wshadow -Wredundant-decls" don't work with Ruby v2.2.0 or v2.3.3)
```

Comment:
The fact that it doesn't work with Ruby 2.2.0 is not relevant if we're not building that. I'd just change it to say `... don't work with Ruby v2.3.3`.